index.html

```diff
@@ -0,0 +1,196 @@
+            src="src/assets/media/icons-logos/ferrari-logo-png.png"
+            width="50"
+            height="50"
+            class="d-inline-block align-top"
+            alt="ferrari logo"
+          />
+        </a>
+        <ul class="d-flex list-unstyled" data-mobile-menu>
+          <li>
+            <a href="about.html" class="my-navbar__link text-decoration-none"
+              >About</a
+            >
+          </li>
+          <li>
+            <a href="#" class="my-navbar__link text-decoration-none">Program</a>
+          </li>
+          <li>
+            <a href="#" class="my-navbar__link text-decoration-none">Join</a>
+          </li>
+          <li>
+            <a href="#" class="my-navbar__link text-decoration-none">Sponsor</a>
+          </li>
+          <li>
+            <a href="#" class="my-navbar__link text-decoration-none">News</a>
+          </li>
+          <li class="mr-3">
+            <a
+              href="https://www.ferrari.com/"
+              class="my-navbar__link bordered-link text-decoration-none"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Ferrari Cars</a
+            >
+          </li>
+          <li>
+            <button type="button" class="x-button" data-toggle-mobile-menu-btn>
+              <i class="fa-sharp fa-solid fa-xmark"></i>
+            </button>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section
+      id="top-page"
+      class="hero-section d-flex flex-column justify-content-left align-items-center"
+    >
+      <div class="hero-section__content container mt-5">
+        <h1 class="hero-section__title display-1">
+          f1 world championship races 2023
+        </h1>
+        <p class="hero-section__intro lead">
+          The winner of the FIA Formula One World Drivers' Championship (WDC) is
+          the most successful Formula One driver over a particular season.
+        </p>
+        <p class="hero-section__date">
+          <time datetime="2023-12-20 12:00">2023.12.20(WED) ~ 23(SAT)</time>
+        </p>
+      </div>
+    </section>
+    <section id="activities" class="d-flex flex-column">
+      <h2 class="section__title text-center display-6 mt-5">Activities</h2>
+      <hr />
+      <div class="activities-container container" data-activities-holder>
+        <!-- activities will be auto injected -->
+      </div>
+      <a
+        href="#"
+        target="_blank"
+        rel="noopener noreferrer"
+        class="desktop-activities-link"
+        >See all the activities</a
+      >
+      <button class="my-btn mobile-buy-tickets-btn" type="button">
+        Buy your exclusive tickets here
+      </button>
+    </section>
+    <section id="upcoming-races" class="d-flex flex-column container">
+      <h2 class="section__title text-center display-6 mt-5">Upcoming Races</h2>
+      <hr />
+      <div
+        class="upcoming-races-container container"
+        data-upcoming-races-holder
+      ></div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add a `chess background shape`(color: optional) to each `Upcoming Race Card` like a [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

![image](https://github.com/ITurres/Ferrari-Racing-Website/assets/63915024/fc53e917-75f3-4cf4-86c1-594f3945e63b)

Reply:
Hi, @mukhammadrizooff! thank you for reviewing my code! 💯 

I forgot to add on the pr notes the reason why the website doesn’t include the chess background, sorry about that!, so the chess BG is part of the images in the [behance](https://summit.cckorea.org/main.html) design, also it matches the pattern on the hero-section background as well, and since this has been changed to a different background image due to different content I left them plain.